lib/fog/storage/google_json.rb

```diff
@@ -29,7 +29,7 @@ class GoogleJSON < Fog::Service
 
       # Version of IAM API used for blob signing, see Fog::Storage::GoogleJSON::Real#iam_signer
       GOOGLE_STORAGE_JSON_IAM_API_VERSION = "v1".freeze
-      GOOGLE_STORAGE_JSON_IAM_API_SCOPE_URLS = %w(https://www.googleapis.com/auth/devstorage.full_control).freeze
+      GOOGLE_STORAGE_JSON_IAM_API_SCOPE_URLS = %w(https://www.googleapis.com/auth/iam).freeze
```

Comment:
@peikk0 Should we use https://www.googleapis.com/auth/cloud-platform instead?

Reply:
https://www.googleapis.com/auth/cloud-platform seems much more permissive. I'm inclined to stick with this?

Reply:
On one hand, `https://www.googleapis.com/auth/iam` is more restrictive and keeps the original intent. On the other hand, access scopes are considered legacy by Google and they recommend using `https://www.googleapis.com/auth/cloud-platform` and proper IAM permissions instead: https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam But changing to that could have security implications for users that still rely on it so it might be safer to stick to `https://www.googleapis.com/auth/iam` for now.

Reply:
👍 Thanks!